### DISCUSSION TITLE
Is windows desktop access without AD only available for enterprise ?

### DISCUSSION THREAD

**Discussion by uaygunn** [gravitational/teleport#21525] [Ideas]
Hello guys does this access type only available for enterprise edition ?

I was so hyped about it but it seems i can not use it.

If not i was not able to make teleport service work when enable windows desktop access in .yaml file what should i do ?

**Comment by zmb3** [comment]
Yes. Windows Access without AD is a preview feature that is only available to enterprise customers at this time.

This may change in the future when the feature moves out of preview. In the meantime, thanks for registering your interest!

**Reply by jones85** [reply]
Do you know of any timescales for this?  I am currently using Apache Guacamole but I am keen to try this as a replacement.  This is for my homelab whereby open source and low cost is key, as well as keeping my infrastructure lean.  Deploying ADDS is not a direction I want to go.

I would have thought, if you wish to gain more traction with this feature, offering it without the prerequisite of AD for your CE addition would increase the uptake of the product. Personally, I would be one of them.

**Reply by zmb3** [reply]
We are working on this now. The plan is for the community edition to support a small number of standalone windows machines in teleport 14.
